TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot/constants/AutonomousConstants.java

```diff
@@ -0,0 +1,18 @@
+package org.firstinspires.ftc.teamcode.robot.constants;
+
+import org.firstinspires.ftc.teamcode.core.lib.pid.PIDController;
+
+public class AutonomousConstants {
+
+    public static final double TICK_TO_CM_CONVERSION_VALUE = -0.0305; //this must be configured with TicksToCmConfiguration opmode before running the robot
+    //public static final double TRACK_WIDTH=0; //distance between parallel wheels viewing the robot from the front
```

Comment:
can we remove this consts?

Reply:
we cannot remove TICK_TO_CM_CONVERSION_VALUE as it is responsible for converting the degrees traveled by the wheels into usable cm, but the ones that are in comment have been removed due to being outdated